art/command_line.py

```diff
@@ -19,8 +19,24 @@
 
 def get_gitlab():
     config = _config.load()
-    return Gitlab(config['gitlab_url'], private_token=config['private_token'])
+    if config['token_type'] == 'private':
+        return Gitlab(config['gitlab_url'], private_token=config['token'])
+    elif config['token_type'] == 'job':
+        return Gitlab(config['gitlab_url'], job_token=config['token'])
+    else:
+        raise Exception("Unknown token type: {}".format(config['token_type']))
+
+def is_using_job_token(gitlab):
+    """Determine if the GitLab client will use a job token to authenticate.
+
+    Job tokens cannot access the full GitLab API. The GitLab client uses a job
```

Comment:
Nit: This statement is true today, but this is GitLab... I think it would be helpful to include a reference to GitLab docs and/or issues here for when things change.

Reply:
Because it's GitLab, I'm not convinced I can even get a stable documentation link, but I will add the current link.

Reply:
Documentation link included in 43cf7dba1ebf573b0114fb3e50cc68b61615c838